# Catalan translation for update-manager
# Copyright (C) 2006
# This file is distributed under the same license as the update-manager package.
# Jordi Irazuzta Cardús <irazuzta@gmail.com>, 2006.
#
#: ../DistUpgrade/DistUpgradeController.py:1556
msgid ""
msgstr ""
"Project-Id-Version: update-manager\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2023-10-26 15:21-0400\n"
"PO-Revision-Date: 2013-05-22 11:00+0000\n"
"Last-Translator: Joan Duran <jodufi@gmail.com>\n"
"Language-Team: Catalan <tradgnome@softcatala.org>\n"
"Language: ca\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Launchpad-Export-Date: 2019-11-15 04:49+0000\n"
"X-Generator: Launchpad (build c597c3229eb023b1e626162d5947141bf7befb13)\n"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:206 ../DistUpgrade/distro.py:461
#, python-format
msgid "Server for %s"
msgstr "Servidor per %s"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:225 ../DistUpgrade/distro.py:237
#: ../DistUpgrade/distro.py:258
msgid "Main server"
msgstr "Servidor principal"

#: ../DistUpgrade/distro.py:267
msgid "Custom servers"
msgstr "Servidors personalitzats"

#: ../DistUpgrade/DistUpgradeCache.py:182
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "Elimina el paquet en mal estat"
msgstr[1] "Elimina els paquets en mal estat"

#: ../DistUpgrade/DistUpgradeCache.py:185
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"El paquet «%s» es troba en un estat inconsistent i s'ha de tornar a "
"instal·lar, però no s'ha trobat l'arxiu per a fer-ho. Voleu eliminar este "
"paquet i continuar?"
msgstr[1] ""
"Els paquets «%s» es troben en un estat inconsistent i s'han de tornar a "
"instal·lar, però no s'ha trobat els arxius per a fer-ho. Voleu eliminar "
"estos paquets i continuar?"

#. FIXME: not ideal error message, but we just reuse a
#. existing one here to avoid a new string
#: ../DistUpgrade/DistUpgradeCache.py:301
msgid "The server may be overloaded"
msgstr "Pot ser que el servidor estiga sobrecarregat"

#: ../DistUpgrade/DistUpgradeCache.py:414
msgid "Broken packages"
msgstr "Paquets trencats"

#: ../DistUpgrade/DistUpgradeCache.py:415
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"El vostre sistema conté paquets trencats que no es poden arreglar amb esta "
"aplicació. Utilitzeu el Synaptic o l'apt-get abans de continuar."

#: ../DistUpgrade/DistUpgradeCache.py:660
#, fuzzy, python-format
msgid "Broken packages after upgrade: %s"
msgstr "Paquets trencats"

#. the most likely problem is the 3rd party pkgs so don't address
#. foreignPkgs and devRelease being True
#: ../DistUpgrade/DistUpgradeCache.py:665
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
" "
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:668
msgid ""
"This was likely caused by:\n"
" * Unofficial software packages not provided by Ubuntu\n"
"Please use the tool 'ppa-purge' from the ppa-purge \n"
"package to remove software from a Launchpad PPA and \n"
"try the upgrade again.\n"
"\n"
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:676
msgid ""
"This was caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
"This is most likely a transient problem, \n"
"please try again later.\n"
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:683
msgid "This is most likely a transient problem, please try again later."
msgstr ""
"Això és probablement un problema transitori, torneu a provar-ho més tard."

#: ../DistUpgrade/DistUpgradeCache.py:686
msgid ""
"If none of this applies, then please report this bug using the command "
"'ubuntu-bug ubuntu-release-upgrader-core' in a terminal. "
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:688
msgid ""
"If you want to investigate this yourself the log files in '/var/log/dist-"
"upgrade' will contain details about the upgrade. Specifically, look at 'main."
"log' and 'apt.log'."
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:694
msgid "Could not calculate the upgrade"
msgstr "No s'ha pogut calcular l'actualització"

#: ../DistUpgrade/DistUpgradeCache.py:751
msgid "Error authenticating some packages"
msgstr "S'ha produït un error en autenticar alguns paquets"

#: ../DistUpgrade/DistUpgradeCache.py:752
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"No s'han pogut autenticar alguns paquets. Potser hi ha un problema temporal "
"amb la xarxa. Podeu provar-ho més tard. A continuació es mostra la llista "
"amb els paquets no autenticats."

#: ../DistUpgrade/DistUpgradeCache.py:773
#, fuzzy, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal deny list."
msgstr ""
"El paquet «%s» està marcat per a eliminar-lo, però apareix a la llista negra "
"de fitxers a eliminar."

#: ../DistUpgrade/DistUpgradeCache.py:779
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "El paquet essencial «%s» està marcat per a ésser eliminat."

#: ../DistUpgrade/DistUpgradeCache.py:788
#, fuzzy, python-format
msgid "Trying to install deny listed version '%s'"
msgstr "S'està intentant instal·lar la versió «%s» de la llista negra"

#: ../DistUpgrade/DistUpgradeCache.py:910
#, python-format
msgid "Can't install '%s'"
msgstr "No s'ha pogut instal·lar «%s»"

#: ../DistUpgrade/DistUpgradeCache.py:911
msgid ""
"It was impossible to install a required package. Please report this as a bug "
"using 'ubuntu-bug ubuntu-release-upgrader-core' in a terminal."
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:922
msgid "Can't guess meta-package"
msgstr "No s'ha pogut conjecturar el metapaquet"

#: ../DistUpgrade/DistUpgradeCache.py:923
#, python-format
msgid ""
"Your system does not contain a %s or %s package and it was not possible to "
"detect which version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"El vostre sistema no conté un paquet %s o %s i no ha estat possible detectar "
"la versió de l'Ubuntu que esteu fent servir.\n"
" Instal·leu un dels paquets mencionats fent servir el synaptic o l'apt-get "
"abans de continuar."

#: ../DistUpgrade/DistUpgradeController.py:145
msgid "Reading cache"
msgstr "S'està llegint la memòria cau"

#: ../DistUpgrade/DistUpgradeController.py:270
msgid "Unable to get exclusive lock"
msgstr "No s'ha pogut obtindre un bloqueig exclusiu"

#: ../DistUpgrade/DistUpgradeController.py:271
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"Normalment això es deu al fet que teniu un altre gestor de paquets en "
"execució (com  ara l'apt-get o l'aptitude) i cal que abans el tanqueu."

#: ../DistUpgrade/DistUpgradeController.py:329
msgid "Upgrading over remote connection not supported"
msgstr "No és possible actualitzar a través d'una connexió remota"

#: ../DistUpgrade/DistUpgradeController.py:330
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. Please try a text mode upgrade with 'do-release-"
"upgrade'.\n"
"\n"
"The upgrade will abort now. Please try without ssh."
msgstr ""
"Esteu duent a terme l'actualització a través d'una connexió SSH remota amb "
"un frontal que no admet esta funció. Intenteu realitzar una actualització en "
"mode text amb l'orde «do-release-upgrade». \n"
"\n"
"S'interromprà l'actualització. Torneu-ho a intentar sense SSH."

#: ../DistUpgrade/DistUpgradeController.py:344
msgid "Continue running under SSH?"
msgstr "Voleu continuar treballant via SSH?"

#: ../DistUpgrade/DistUpgradeController.py:345
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, an additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"Pareix ser que esta sessió s'està executant amb SSH. Actualment no és "
"recomanable realitzar una actualització a través d'SSH, atès que en cas de "
"fallada la recuperació és més difícil.\n"
"\n"
"Si continueu, s'iniciarà un dimoni addicional al port «%s».\n"
"Voleu continuar?"

#: ../DistUpgrade/DistUpgradeController.py:359
msgid "Starting additional sshd"
msgstr "S'està iniciant un sshd addicional"

#: ../DistUpgrade/DistUpgradeController.py:360
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"Per facilitar la recuperació en cas de fallada, s'iniciarà un sshd "
"addicional al port «%s». Si alguna cosa anés malament amb l'ssh en ús, podeu "
"fer servir l'addicional.\n"

#: ../DistUpgrade/DistUpgradeController.py:368
#, python-format
msgid ""
"If you run a firewall, you may need to temporarily open this port. As this "
"is potentially dangerous it's not done automatically. You can open the port "
"with e.g.:\n"
"'%s'"
msgstr ""
"Si feu servir un tallafocs, necessitareu obrir temporalment este port. Atès "
"que això és potencialment perillós, no es fa automàticament. Per exemple, "
"podeu obrir el port amb:\n"
"«%s»"

#: ../DistUpgrade/DistUpgradeController.py:446
#: ../DistUpgrade/DistUpgradeController.py:470
msgid "Can not upgrade"
msgstr "No es pot actualitzar"

#: ../DistUpgrade/DistUpgradeController.py:447
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr "Esta eina no permet l'actualització de «%s» a «%s»."

#: ../DistUpgrade/DistUpgradeController.py:471
msgid ""
"Your python3 install is corrupted. Please fix the '/usr/bin/python3' symlink."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:500
#, python-format
msgid "Can not write to '%s'"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:501
#, python-format
msgid ""
"Its not possible to write to the system directory '%s' on your system. The "
"upgrade can not continue.\n"
"Please make sure that the system directory is writable."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:720
#, python-format
msgid "Not for humans during development stage of release %s"
msgstr ""
"No està pensat per a éssers humans mentre %s està en fase de desenvolupament"

#: ../DistUpgrade/DistUpgradeController.py:821
#, python-format
msgid "disabled on upgrade to %s"
msgstr "desactivat en actualitzar a %s"

#: ../DistUpgrade/DistUpgradeController.py:1221
#, fuzzy
msgid "No valid source entry found"
msgstr "No s'ha trobat cap rèplica vàlida"

#: ../DistUpgrade/DistUpgradeController.py:1222
#: ../DistUpgrade/DistUpgradeController.py:1290
#, python-format
msgid ""
"While scanning your repository information no entry about %s could be "
"found.\n"
"\n"
"An upgrade might not succeed.\n"
"\n"
"Do you want to continue anyway?"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1232
#: ../DistUpgrade/DistUpgradeController.py:1300
msgid "No valid mirror found"
msgstr "No s'ha trobat cap rèplica vàlida"

#: ../DistUpgrade/DistUpgradeController.py:1233
#: ../DistUpgrade/DistUpgradeController.py:1301
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run an internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:1253
#: ../DistUpgrade/DistUpgradeController.py:1321
msgid "Generate default sources?"
msgstr "Voleu generar les fonts per defecte?"

#: ../DistUpgrade/DistUpgradeController.py:1254
#, fuzzy, python-format
msgid ""
"After scanning your 'ubuntu.sources' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"No s'ha trobat cap entrada vàlida per a «%s» en analitzar el fitxer «sources."
"list».\n"
"\n"
"Voleu que s'afigen entrades per a «%s»? Si trieu que «No», es cancel·larà "
"l'actualització."

#: ../DistUpgrade/DistUpgradeController.py:1272
#: ../DistUpgrade/DistUpgradeController.py:1373
msgid "Third party sources disabled"
msgstr "S'han desactivat les fonts de tercers"

#: ../DistUpgrade/DistUpgradeController.py:1273
#: ../DistUpgrade/DistUpgradeController.py:1374
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"S'han desactivat algunes entrades de tercers. Les podeu reactivar després de "
"l'actualització des de l'eina «software-properties» en el vostre gestor de "
"paquets."

#: ../DistUpgrade/DistUpgradeController.py:1289
msgid "No valid sources.list entry found"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1322
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"No s'ha trobat cap entrada vàlida per a «%s» en analitzar el fitxer «sources."
"list».\n"
"\n"
"Voleu que s'afigen entrades per a «%s»? Si trieu que «No», es cancel·larà "
"l'actualització."

#: ../DistUpgrade/DistUpgradeController.py:1357
#: ../DistUpgrade/DistUpgradeController.py:1363
msgid "Repository information invalid"
msgstr "La informació dels dipòsits no és vàlida"

#: ../DistUpgrade/DistUpgradeController.py:1358
msgid ""
"Upgrading the repository information resulted in a invalid file so a bug "
"reporting process is being started."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1364
msgid ""
"Upgrading the repository information resulted in a invalid file. To report a "
"bug install apport and then execute 'apport-bug ubuntu-release-upgrader'."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1416
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "Paquet en estat inconsistent"
msgstr[1] "Paquets en estat inconsistent"

#: ../DistUpgrade/DistUpgradeController.py:1419
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"El paquet «%s» es troba en un estat inconsistent i s'ha de tornar a "
"instal·lar, però no s'ha trobat l'arxiu per a fer-ho. Torneu-lo a instal·lar "
"manualment o suprimiu-lo del sistema."
msgstr[1] ""
"Els paquets «%s» es troben en un estat inconsistent i s'han de tornar a "
"instal·lar, però no s'ha trobat els arxius per a fer-ho. Torneu-los a "
"instal·lar manualment o suprimiu-los del sistema."

#: ../DistUpgrade/DistUpgradeController.py:1480
msgid "Error during update"
msgstr "S'ha produït un error en l'actualització"

#: ../DistUpgrade/DistUpgradeController.py:1481
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"S'ha produït un error mentre s'actualitzava el vostre sistema. Normalment "
"això es degut a problemes de xarxa. Comproveu la vostra connexió de xarxa i "
"torneu a intentar-ho."

#: ../DistUpgrade/DistUpgradeController.py:1524
#: ../DistUpgrade/DistUpgradeController.py:1529
msgid "EFI System Partition (ESP) not usable"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1525
msgid ""
"Your EFI System Partition (ESP) is not mounted at /boot/efi. Please ensure "
"that it is properly configured and try again."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1530
msgid ""
"The EFI System Partition (ESP) mounted at /boot/efi is not writable. Please "
"mount this partition read-write and try again."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1537
msgid "Not enough free disk space"
msgstr "No disposeu de suficient espai al disc"

#. TRANSLATORS: you can change the order of the sentence,
#. make sure to keep all {str_*} string untranslated.
#: ../DistUpgrade/DistUpgradeController.py:1540
#, python-brace-format
msgid ""
"The upgrade has aborted. The upgrade needs a total of {str_total} free space "
"on disk '{str_dir}'. Please free at least an additional {str_needed} of disk "
"space on '{str_dir}'. {str_remedy}"
msgstr ""

#. specific ways to resolve lack of free space
#: ../DistUpgrade/DistUpgradeController.py:1545
msgid ""
"Remove temporary packages of former installations using 'sudo apt clean'."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1547
msgid ""
"You can remove old kernels using 'sudo apt autoremove' and you could also "
"set COMPRESS=xz in /etc/initramfs-tools/initramfs.conf to reduce the size of "
"your initramfs."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1552
msgid ""
"Empty your trash and remove temporary packages of former installations using "
"'sudo apt-get clean'."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1555
msgid "Reboot to clean up files in /tmp."
msgstr ""

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:1594
#: ../DistUpgrade/DistUpgradeController.py:2454
#: ../DistUpgrade/DistUpgradeController.py:2506
msgid "Calculating the changes"
msgstr "S'estan calculant els canvis"

#. ask the user
#: ../DistUpgrade/DistUpgradeController.py:1638
msgid "Do you want to start the upgrade?"
msgstr "Voleu iniciar l'actualització?"

#: ../DistUpgrade/DistUpgradeController.py:1657
#, python-format
msgid ""
"Livepatch security updates are not available for Ubuntu %s. If you upgrade, "
"Livepatch will turn off."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1729
msgid "Upgrade canceled"
msgstr "S'ha cancel·lat l'actualització"

#: ../DistUpgrade/DistUpgradeController.py:1730
msgid ""
"The upgrade will cancel now and the original system state will be restored. "
"You can resume the upgrade at a later time."
msgstr ""
"Ara es cancel·larà l'actualització i es restaurarà a l'estat original del "
"sistema. Podeu continuar l'actualització més tard."

#: ../DistUpgrade/DistUpgradeController.py:1736
#: ../DistUpgrade/DistUpgradeController.py:1881
msgid "Could not download the upgrades"
msgstr "No s'han pogut baixar les actualitzacions"

#: ../DistUpgrade/DistUpgradeController.py:1737
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far have been kept."
msgstr ""
"S'ha interromput l'actualització. Comproveu la connexió a Internet o els "
"suports d'instal·lació i torneu-ho a provar. S'han mantingut tots els "
"fitxers baixats fins ara."

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1830
#: ../DistUpgrade/DistUpgradeController.py:1869
#: ../DistUpgrade/DistUpgradeController.py:1986
msgid "Error during commit"
msgstr "S'ha produït un error durant l'enviament"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1832
#: ../DistUpgrade/DistUpgradeController.py:1871
#: ../DistUpgrade/DistUpgradeController.py:2035
msgid "Restoring original system state"
msgstr "S'està restaurant l'estat original del sistema"

#: ../DistUpgrade/DistUpgradeController.py:1833
#: ../DistUpgrade/DistUpgradeController.py:1848
#: ../DistUpgrade/DistUpgradeController.py:1872
msgid "Could not install the upgrades"
msgstr "No s'han pogut instal·lar les actualitzacions"

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1838
msgid ""
"The upgrade has aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"S'ha interromput l'actualització. Pot ser que el vostre sistema haja quedat "
"en un estat inestable. Ara s'executarà un procés de recuperació (dpkg --"
"configure -a)."

#: ../DistUpgrade/DistUpgradeController.py:1843
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug in a browser at http://bugs.launchpad.net/ubuntu/"
"+source/ubuntu-release-upgrader/+filebug and attach the files in /var/log/"
"dist-upgrade/ to the bug report.\n"
"%s"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1882
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr ""
"S'ha interromput l'actualització. Verifiqueu la vostra connexió a Internet o "
"el suport d'instal·lació i torneu-ho a intentar. "

#: ../DistUpgrade/DistUpgradeController.py:1910
msgid "Searching for obsolete software"
msgstr "S'està cercant programari obsolet"

#: ../DistUpgrade/DistUpgradeController.py:1974
msgid "Remove obsolete packages?"
msgstr "Voleu suprimir els paquets obsolets?"

#: ../DistUpgrade/DistUpgradeController.py:1975
#: ../data/gtkbuilder/DistUpgrade.ui.h:8
msgid "_Keep"
msgstr "_Manté"

#: ../DistUpgrade/DistUpgradeController.py:1975
msgid "_Remove"
msgstr "Sup_rimeix"

#: ../DistUpgrade/DistUpgradeController.py:1987
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr ""
"S'ha produït un error durant el procés de neteja. Vegeu el següent missatge "
"per a més informació. "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:2073
msgid "Required depends is not installed"
msgstr "Les dependències requerides no estan instal·lades"

#: ../DistUpgrade/DistUpgradeController.py:2074
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "La dependència requerida «%s» no està instal·lada. "

#. coherence check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:2307
#: ../DistUpgrade/DistUpgradeController.py:2402
msgid "Checking package manager"
msgstr "S'està comprovant el gestor de paquets"

#: ../DistUpgrade/DistUpgradeController.py:2313
#: ../DistUpgrade/DistUpgradeController.py:2319
msgid "Preparing the upgrade failed"
msgstr "S'ha produït un error en la preparació de l'actualització"

#: ../DistUpgrade/DistUpgradeController.py:2314
msgid ""
"Preparing the system for the upgrade failed so a bug reporting process is "
"being started."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:2320
msgid ""
"Preparing the system for the upgrade failed. To report a bug install apport "
"and then execute 'apport-bug ubuntu-release-upgrader'."
msgstr ""
"Ha fallat la preparació de l'actualització del sistema. Per informar d'un "
"error, instal·leu l'apport i executeu «apport-bug ubuntu-release-upgrader»"

#: ../DistUpgrade/DistUpgradeController.py:2337
#: ../DistUpgrade/DistUpgradeController.py:2347
msgid "Getting upgrade prerequisites failed"
msgstr "S'ha produït un error en obtindre els prerequisits de l'actualització"

#: ../DistUpgrade/DistUpgradeController.py:2338
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Additionally, a bug reporting process is being started."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:2348
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"To report a bug install apport and then execute 'apport-bug ubuntu-release-"
"upgrader'."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:2382
msgid "Updating repository information"
msgstr "S'està actualitzant la informació dels dipòsits"

#: ../DistUpgrade/DistUpgradeController.py:2431
msgid "Invalid package information"
msgstr "La informació dels paquets no és valida"

#: ../DistUpgrade/DistUpgradeController.py:2432
#, python-format
msgid ""
"After updating your package information, the essential package '%s' could "
"not be located. This may be because you have no official mirrors listed in "
"your software sources, or because of excessive load on the mirror you are "
"using. See /etc/apt/sources.list for the current list of configured software "
"sources.\n"
"In the case of an overloaded mirror, you may want to try the upgrade again "
"later."
msgstr ""
"Després d'actualitzar la informació dels paquets, no s'ha pogut localitzar "
"el paquet essencial «%s». Això pot ser degut a que no teniu cap rèplica "
"oficial al llistat de fonts de programari o perquè hi ha una carrega "
"excessiva en la rèplica que esteu utilitzant. Vegeu /etc/apt/sources.list "
"per veure la llista actual de fonts de programari.\n"
"En cas de rèplica amb sobrecàrrega, podeu intentar actualitzar més tard."

#: ../DistUpgrade/DistUpgradeController.py:2461
#: ../DistUpgrade/DistUpgradeController.py:2572
msgid "Fetching"
msgstr "S'està recollint"

#: ../DistUpgrade/DistUpgradeController.py:2468
#: ../DistUpgrade/DistUpgradeController.py:2491
#: ../DistUpgrade/DistUpgradeController.py:2521
#: ../DistUpgrade/DistUpgradeController.py:2576
msgid "Upgrading"
msgstr "S'està actualitzant"

#: ../DistUpgrade/DistUpgradeController.py:2470
msgid "Upgrade infeasible"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:2471
msgid ""
"The upgrade could not be completed, there were errors during the upgrade "
"process."
msgstr ""

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:2494
#: ../DistUpgrade/DistUpgradeController.py:2510
msgid "Upgrade incomplete"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:2495
#: ../DistUpgrade/DistUpgradeController.py:2511
msgid ""
"The upgrade has partially completed but there were errors during the upgrade "
"process."
msgstr ""

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:2526
#: ../DistUpgrade/DistUpgradeController.py:2578
#: ../DistUpgrade/DistUpgradeController.py:2584
#: ../DistUpgrade/DistUpgradeController.py:2595
msgid "Upgrade complete"
msgstr "S'ha completat l'actualització"

#: ../DistUpgrade/DistUpgradeController.py:2527
#: ../DistUpgrade/DistUpgradeController.py:2579
#: ../DistUpgrade/DistUpgradeController.py:2585
msgid ""
"The upgrade has completed but there were errors during the upgrade process."
msgstr ""
"S'ha completat l'actualització, però s'han produït errors durant este procés."

#: ../DistUpgrade/DistUpgradeController.py:2542
msgid "System upgrade is complete."
msgstr "S'ha completat l'actualització del sistema."

#: ../DistUpgrade/DistUpgradeController.py:2596
msgid "The partial upgrade was completed."
msgstr "L'actualització parcial s'ha completat."

#: ../DistUpgrade/DistUpgradeController.py:2617
msgid "Lock screen disabled"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:2618
msgid ""
"Your lock screen has been disabled and will remain disabled during the "
"upgrade."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:2626
msgid "Unable to disable lock screen"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:2627
msgid ""
"It is highly recommended that the lock screen be disabled during the upgrade "
"to prevent later issues. Please ensure your screen lock is disabled before "
"continuing."
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcher.py:107
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:170
msgid "Could not find the release notes"
msgstr "No s'han trobat les notes de la versió"

#: ../DistUpgrade/DistUpgradeFetcher.py:108
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:171
msgid "The server may be overloaded. "
msgstr "El servidor potser està sobrecarregat. "

#: ../DistUpgrade/DistUpgradeFetcher.py:120
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:175
msgid "Could not download the release notes"
msgstr "No s'han pogut baixar les notes de la versió"

#: ../DistUpgrade/DistUpgradeFetcher.py:121
#: ../DistUpgrade/DistUpgradeFetcherCore.py:77
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:176
msgid "Please check your internet connection."
msgstr "Comproveu la vostra connexió a Internet"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:72
#, fuzzy
msgid "Could not find the release announcement"
msgstr "No s'han trobat les notes de la versió"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:73
#, fuzzy
msgid "The server may be overloaded."
msgstr "El servidor potser està sobrecarregat. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:76
#, fuzzy
msgid "Could not download the release announcement"
msgstr "No s'han pogut baixar les notes de la versió"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:82
#: ../DistUpgrade/DistUpgradeViewText.py:215
#: ../DistUpgrade/DistUpgradeViewText.py:253
#: ../DistUpgrade/DistUpgradeViewText.py:261
msgid "Continue [yN] "
msgstr "Continua [sN] "

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeFetcherCore.py:84
#: ../DistUpgrade/DistUpgradeViewText.py:219
#: ../DistUpgrade/DistUpgradeViewText.py:264
msgid "y"
msgstr "s"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:101
#, python-format
msgid "authenticate '%(file)s' against '%(signature)s' "
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcherCore.py:127
#, python-format
msgid "extracting '%s'"
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcherCore.py:148
#: ../DistUpgrade/DistUpgradeFetcherCore.py:149
msgid "Could not run the upgrade tool"
msgstr "No es pot executar l'eina d'actualització"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:150
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug "
"using the command 'ubuntu-bug ubuntu-release-upgrader-core'."
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcherCore.py:227
msgid "Upgrade tool signature"
msgstr "Signatura de l'eina d'actualització"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:234
msgid "Upgrade tool"
msgstr "Eina d'actualització"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:268
msgid "Failed to fetch"
msgstr "Ha fallat la baixada"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:269
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr ""
"Ha fallat l'obtenció de l'actualització. Pot ser que hi haja algun problema "
"a la xarxa. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:273
msgid "Authentication failed"
msgstr "Ha fallat l'autenticació"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:274
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr ""
"Ha fallat l'autenticació de l'actualització. Pot ser que hi haja algun "
"problema a la xarxa o al servidor. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:279
msgid "Failed to extract"
msgstr "Ha fallat l'extracció"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:280
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr ""
"Ha fallat l'extracció de l'actualització. Pot ser que hi haja algun problema "
"a la xarxa o al servidor. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:285
msgid "Verification failed"
msgstr "Ha fallat la verificació"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:286
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr ""
"Ha fallat la verificació de l'actualització. Pot ser que hi haja algun "
"problema a la xarxa o al servidor. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:300
#: ../DistUpgrade/DistUpgradeFetcherCore.py:306
msgid "Can not run the upgrade"
msgstr "No es pot executar l'actualització"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:301
msgid ""
"This usually is caused by a system where /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""
"Normalment això està provocat per un sistema on /tmp s'ha muntat com a no "
"executable. Torneu-lo a muntar sense l'opció de no executable i torneu a "
"executar l'actualització."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:307
#, python-format
msgid "The error message is '%s'."
msgstr "El missatge d'error és «%s»."

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:151
msgid "&Upgrade"
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:154
#: ../DistUpgrade/DistUpgradeViewKDE.py:690
msgid "&Cancel"
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:156
#: ../data/gtkbuilder/ReleaseNotes.ui.h:1
msgid "Release Notes"
msgstr "Notes de la versió"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:191
msgid "Upgrade"
msgstr "Actualització"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:207
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:222
msgid "Downloading additional package files..."
msgstr "S'estan baixant els fitxers de paquet addicionals..."

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:224
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "Fitxer %s de %s a %s B/s"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:228
#, python-format
msgid "File %s of %s"
msgstr "Fitxer %s de %s"

#. print("mediaChange %s %s" % (medium, drive))
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:235
#: ../DistUpgrade/DistUpgradeViewGtk3.py:127
#: ../DistUpgrade/DistUpgradeViewKDE.py:302
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Inseriu «%s» a la unitat «%s»"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:237
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:242
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:243
#: ../DistUpgrade/DistUpgradeViewKDE.py:303
msgid "Media Change"
msgstr "Canvi de suport"

#: ../DistUpgrade/DistUpgradeQuirks.py:241
msgid "Your graphics hardware may not be fully supported in Ubuntu 14.04."
msgstr ""
"Pot ser que el vostre maquinari de gràfics no funcione bé amb l'Ubuntu 14.04."

#: ../DistUpgrade/DistUpgradeQuirks.py:243
msgid ""
"Running the 'unity' desktop environment is not fully supported by your "
"graphics hardware. You will maybe end up in a very slow environment after "
"the upgrade. Our advice is to keep the LTS version for now. For more "
"information see https://wiki.ubuntu.com/X/Bugs/"
"UpdateManagerWarningForUnity3D Do you still want to continue with the "
"upgrade?"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:267
msgid "Your graphics hardware may not be fully supported in Ubuntu 12.04 LTS."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:269
msgid ""
"The support in Ubuntu 12.04 LTS for your Intel graphics hardware is limited "
"and you may encounter problems after the upgrade. For more information see "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx Do you want to "
"continue with the upgrade?"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:289
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr ""
"L'actualització pot reduir els efectes d'escriptori i el rendiment en jocs i "
"altres programes que facen un ús exhaustiu de processament gràfic."

#: ../DistUpgrade/DistUpgradeQuirks.py:293
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04 "
"LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Este ordinador utilitza el controlador gràfic «fglrx» d'AMD. No hi ha cap "
"versió disponible d'este controlador que funcione amb el vostre maquinari a "
"l'Ubuntu 10.04 LTS.\n"
"\n"
"Voleu continuar?"

#: ../DistUpgrade/DistUpgradeQuirks.py:332
#: ../DistUpgrade/DistUpgradeQuirks.py:401
msgid "Sorry, no more upgrades for this system"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:333
#, python-format
msgid ""
"There will not be any further Ubuntu releases for this system's 'i386' "
"architecture.\n"
"\n"
"Updates for Ubuntu %s will continue until %s."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:348
msgid "No ARMv6 CPU"
msgstr "No és una CPU ARMv6"

#: ../DistUpgrade/DistUpgradeQuirks.py:349
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"El vostre sistema utilitza una CPU ARM que és més antiga que l'arquitectura "
"ARMv6. Tots els paquets del Karmic s'han creat amb optimitzacions que "
"requereixen l'ARMv6 com a arquitectura mínima. No és possible actualitzar el "
"vostre sistema a una versió nova de l'Ubuntu amb este maquinari."

#: ../DistUpgrade/DistUpgradeQuirks.py:369
msgid ""
"Sorry, this storage driver is not supported in kernels for newer releases"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:371
#, python-format
msgid ""
"There will not be any further Ubuntu releases that provide kernel support "
"for the aufs storage driver.\n"
"\n"
"Please ensure that none of your containers are using the aufs storage "
"driver, remove the directory %s and try again."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:402
#, python-format
msgid ""
"There will not be any further Ubuntu releases for this system's POWER8 "
"processor.\n"
"\n"
"Updates for Ubuntu %s will continue until %s."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:420
msgid "No init available"
msgstr "No hi ha cap sistema d'inicialització disponible"

#: ../DistUpgrade/DistUpgradeQuirks.py:421
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 10.04 LTS cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""
"Pareix ser que el vostre sistema és un entorn virtual sense un dimoni "
"d'inicialització «init», com ara un servidor virtual de Linux (Linux-"
"VServer). L'Ubuntu 10.04 no pot funcionar en este tipus d'entorn - abans cal "
"fer una actualització de la configuració de la màquina virtual.\n"
"\n"
"Segur que voleu continuar?"

#: ../DistUpgrade/DistUpgradeQuirks.py:462
msgid "The Robot Operating System (ROS) is installed"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:463
msgid ""
"It appears that ROS is currently installed. Each ROS release is very strict "
"about the versions of Ubuntu it supports, and Ubuntu upgrades can fail if "
"that guidance isn't followed. Before continuing, please either uninstall "
"ROS, or ensure the ROS release you have installed supports the version of "
"Ubuntu to which you're upgrading.\n"
"\n"
"For ROS 1 releases, refer to REP 3:\n"
"https://www.ros.org/reps/rep-0003.html\n"
"\n"
"For ROS 2 releases, refer to REP 2000:\n"
"https://www.ros.org/reps/rep-2000.html\n"
"\n"
"Are you sure you want to continue?"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:656
msgid "Connection to the Snap Store failed"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:657
msgid ""
"You have the package lxd installed but your system is unable to reach the "
"Snap Store. lxd is now provided via a snap and the release upgrade will fail "
"if snapd is not functional. Please make sure you're connected to the "
"Internet and update any firewall or proxy settings as needed so that you can "
"reach api.snapcraft.io. If you are an enterprise with a firewall setup you "
"may want to configure a Snap Store proxy."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:672
msgid "Connection to Snap Store failed"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:673
msgid ""
"Your system does not have a connection to the Snap Store. For the best "
"upgrade experience make sure that your system can connect to api.snapcraft."
"io.\n"
"Do you still want to continue with the upgrade?"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:682
msgid "Outdated snapd package"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:683
msgid ""
"Your system does not have the latest version of snapd. Please update the "
"version of snapd on your system to improve the upgrade experience.\n"
"Do you still want to continue with the upgrade?"
msgstr ""

#. now perform direct API calls to the store, requesting size
#. information for each of the snaps needing installation
#: ../DistUpgrade/DistUpgradeQuirks.py:708
msgid "Calculating snap size requirements"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:739
msgid "Processing snap replacements"
msgstr ""

#. TODO: This status should be updated, but the time of
#. this change to snap switch is post-translation freeze.
#: ../DistUpgrade/DistUpgradeQuirks.py:747
#, python-format
msgid "refreshing snap %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:751
#, python-format
msgid "removing snap %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:754
#, python-format
msgid "installing snap %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:783
msgid "PAE not enabled"
msgstr "El PAE no està habilitat"

#: ../DistUpgrade/DistUpgradeQuirks.py:784
msgid ""
"Your system uses a CPU that does not have PAE enabled. Ubuntu only supports "
"non-PAE systems up to Ubuntu 12.04. To upgrade to a later version of Ubuntu, "
"you must enable PAE (if this is possible) see:\n"
"http://help.ubuntu.com/community/EnablingPAE"
msgstr ""
"El sistema incorpora una CPU que no té el PAE habilitat. L'Ubuntu funciona "
"amb sistemes sense PAE només fins la versió 12.04. Per actualitzar a una "
"versió posterior de l'Ubuntu, haureu d'habilitar el PAE (si és possible). "
"Consulteu la documentació següent per obtindre més informació:\n"
"http://help.ubuntu.com/community/EnablingPAE"

#: ../DistUpgrade/DistUpgradeQuirks.py:1080
msgid "universe component not enabled"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:1081
#, python-format
msgid ""
"You have the package %s installed which is a python2 package. python2 has "
"been deprecated in Ubuntu 20.04 LTS and is now available from the universe "
"component of the Ubuntu archive. To ensure any existing third party code you "
"have installed continues to work the release upgrade process would install "
"%s for you. However, the universe component of the archive is not enabled on "
"this system. For the upgrade process to proceed you should either enable the "
"universe component in /etc/apt/sources.list or remove %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:1193
msgid "Checking for installed snaps"
msgstr ""

#: ../DistUpgrade/DistUpgradeMain.py:62
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"Utilitzeu un frontal. Actualment es disposa de: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:65
msgid "*DEPRECATED* this option will be ignored"
msgstr "*OBSOLET* esta opció s'ignorarà"

#: ../DistUpgrade/DistUpgradeMain.py:68
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr ""
"Realitza només una actualització parcial (sense reescriure el sources.list)"

#: ../DistUpgrade/DistUpgradeMain.py:71
msgid "Disable GNU screen support"
msgstr "Inhabilita la compatibilitat amb el GNU screen"

#: ../DistUpgrade/DistUpgradeMain.py:73
msgid "Set datadir"
msgstr "Estableix el «datadir» (directori de dades)"

#: ../DistUpgrade/DistUpgradeMain.py:76
msgid "Upgrade to the development release"
msgstr "Actualitza a la versió de desenvolupament"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:148
#: ../DistUpgrade/DistUpgradeViewKDE.py:316
msgid "Fetching is complete"
msgstr "S'ha completat el recull"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:163
#: ../DistUpgrade/DistUpgradeViewKDE.py:332
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "S'està obtenint el fitxer %li de %li a %s B/s"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:165
#: ../DistUpgrade/DistUpgradeViewGtk3.py:327
#: ../DistUpgrade/DistUpgradeViewKDE.py:333
#: ../DistUpgrade/DistUpgradeViewKDE.py:483
#, python-format
msgid "About %s remaining"
msgstr "Queden %s"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:168
#: ../DistUpgrade/DistUpgradeViewKDE.py:335
#, python-format
msgid "Fetching file %li of %li"
msgstr "S'està obtenint el fitxer %li de %li"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk3.py:200
#: ../DistUpgrade/DistUpgradeViewKDE.py:373
msgid "Applying changes"
msgstr "S'estan aplicant els canvis"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk3.py:226
#: ../DistUpgrade/DistUpgradeViewKDE.py:386
msgid "dependency problems - leaving unconfigured"
msgstr "problemes de dependències - es deixarà sense configurar"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:231
#: ../DistUpgrade/DistUpgradeViewKDE.py:388
#, python-format
msgid "Could not install '%s'"
msgstr "No s'ha pogut instal·lar «%s»"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:232
#: ../DistUpgrade/DistUpgradeViewKDE.py:389
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr ""
"L'actualització continuarà, però pot ser que el paquet «%s» no siga "
"funcional. Hauríeu de considerar l'enviament d'un informe d'error sobre este "
"fet."

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk3.py:249
#: ../DistUpgrade/DistUpgradeViewKDE.py:411
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"Voleu reemplaçar el fitxer de\n"
"configuració personalitzat «%s»?"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:250
#: ../DistUpgrade/DistUpgradeViewKDE.py:412
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""
"Perdreu tots els canvis realitzats en el fitxer de configuració si trieu "
"reemplaçar-lo per una versió més nova."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:270
#: ../DistUpgrade/DistUpgradeViewKDE.py:435
msgid "The 'diff' command was not found"
msgstr "No s'ha trobat l'orde «diff»"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:542
#: ../DistUpgrade/DistUpgradeViewText.py:140
msgid "A fatal error occurred"
msgstr "S'ha produït un error greu"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:543
msgid ""
"Please report this as a bug (if you haven't already) and include the files /"
"var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Informeu d'això com un error (si no és que ja ho heu fet) i adjunteu els "
"fitxers /var/log/dist-upgrade/main.log i /var/log/dist-upgrade/apt.log al "
"vostre informe. S'ha interromput l'actualització.\n"
"S'ha guardat el vostre fitxer sources.list original a /etc/apt/sources.list."
"distUpgrade."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:573
msgid "Ctrl-c pressed"
msgstr "S'ha premut Ctrl+C"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:574
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr ""
"Això interromprà l'operació, amb la qual cosa pot ser que es malmeti el "
"sistema. Esteu segur que ho voleu fer?"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk3.py:713
msgid "To prevent data loss close all open applications and documents."
msgstr ""
"Per a evitar una possible pèrdua de dades, tanqueu tots els documents i "
"aplicacions."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:727
#, python-format
msgid "No longer supported by Canonical (%s)"
msgstr "Ja no mantinguts per Canonical (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:728
#, python-format
msgid "<b>Downgrade (%s)</b>"
msgstr "<b>Es desactualitzaran (%s)</b>"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:729
#, python-format
msgid "Remove (%s)"
msgstr "Se suprimiran (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:730
#, python-format
msgid "No longer needed (%s)"
msgstr "Ja no són necessaris (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:731
#, python-format
msgid "Install (%s)"
msgstr "S'instal·laran (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:732
#, python-format
msgid "Upgrade (%s)"
msgstr "S'actualitzaran (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:780
msgid "Cancel"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk3.py:781
#: ../DistUpgrade/DistUpgradeViewKDE.py:914
msgid "Continue"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewKDE.py:447
msgid "Show Difference >>>"
msgstr "Mostra les diferències >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:450
msgid "<<< Hide Difference"
msgstr "<<< Oculta les diferències"

#: ../DistUpgrade/DistUpgradeViewKDE.py:676
msgid "Error"
msgstr "S'ha produït un error"

#: ../DistUpgrade/DistUpgradeViewKDE.py:694
#: ../DistUpgrade/DistUpgradeViewKDE.py:935
msgid "&Close"
msgstr "&Tanca"

#: ../DistUpgrade/DistUpgradeViewKDE.py:728
msgid "Show Terminal >>>"
msgstr "Mostra el terminal >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:731
msgid "<<< Hide Terminal"
msgstr "<<< Oculta el terminal"

#: ../DistUpgrade/DistUpgradeViewKDE.py:804
msgid "Information"
msgstr "Informació"

#: ../DistUpgrade/DistUpgradeViewKDE.py:852
msgid "&Start Upgrade"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewKDE.py:854
#: ../DistUpgrade/DistUpgradeViewKDE.py:898
#: ../DistUpgrade/DistUpgradeViewKDE.py:901
#: ../data/gtkbuilder/DistUpgrade.ui.h:7
msgid "Details"
msgstr "Detalls"

#: ../DistUpgrade/DistUpgradeViewKDE.py:876
#, python-format
msgid "No longer supported %s"
msgstr "Ja no es mantenen %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:878
#, python-format
msgid "Remove %s"
msgstr "Suprimeix %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:880
#: ../DistUpgrade/DistUpgradeViewText.py:239
#, python-format
msgid "Remove (was auto installed) %s"
msgstr "Suprimeix %s (s'havia instal·lat automàticament)"

#: ../DistUpgrade/DistUpgradeViewKDE.py:882
#, python-format
msgid "Install %s"
msgstr "Instal·la %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:884
#, python-format
msgid "Upgrade %s"
msgstr "Actualitza %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:931
#: ../DistUpgrade/DistUpgradeViewText.py:292
msgid "Restart required"
msgstr "Cal que reinicieu el sistema"

#: ../DistUpgrade/DistUpgradeViewKDE.py:931
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr "<b><big>Reinicieu el sistema per a completar l'actualització</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:934
#: ../data/gtkbuilder/DistUpgrade.ui.h:14
msgid "_Restart Now"
msgstr "_Reinicia"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:952
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>Voleu cancel·lar l'actualització en curs?</big></b>\n"
"\n"
"El sistema pot quedar inusable si la cancel·leu. És molt recomanable "
"continuar amb l'actualització."

#: ../DistUpgrade/DistUpgradeViewKDE.py:956
msgid "Cancel Upgrade?"
msgstr "Voleu cancel·lar l'actualització?"

#: ../DistUpgrade/DistUpgradeView.py:82
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li dia"
msgstr[1] "%li dies"

#: ../DistUpgrade/DistUpgradeView.py:84
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li hora"
msgstr[1] "%li hores"

#: ../DistUpgrade/DistUpgradeView.py:86
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li minut"
msgstr[1] "%li minuts"

#: ../DistUpgrade/DistUpgradeView.py:87
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li segon"
msgstr[1] "%li segons"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:103
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:121
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#: ../DistUpgrade/DistUpgradeView.py:172
#, fuzzy, python-format
msgid ""
"This download will take about %s with a 40Mbit connection and about %s with "
"a 5Mbit connection."
msgstr ""
"La baixada trigarà aproximadament %s amb una connexió ADSL d'1Mb, i "
"aproximadament %s amb un mòdem de 56k"

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:176
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "La baixada trigarà aproximadament %s amb la vostra connexió. "

#. Declare these translatable strings from the .ui files here so that
#. xgettext picks them up.
#: ../DistUpgrade/DistUpgradeView.py:270 ../data/gtkbuilder/DistUpgrade.ui.h:22
msgid "Preparing to upgrade"
msgstr "S'està preparant l'actualització"

#: ../DistUpgrade/DistUpgradeView.py:271
msgid "Getting new software channels"
msgstr "S'estan obtenint canals de programari nous"

#: ../DistUpgrade/DistUpgradeView.py:272 ../data/gtkbuilder/DistUpgrade.ui.h:24
msgid "Getting new packages"
msgstr "S'estan obtenint els paquets nous"

#: ../DistUpgrade/DistUpgradeView.py:273 ../data/gtkbuilder/DistUpgrade.ui.h:27
msgid "Installing the upgrades"
msgstr "S'estan instal·lant les actualitzacions"

#: ../DistUpgrade/DistUpgradeView.py:274 ../data/gtkbuilder/DistUpgrade.ui.h:26
msgid "Cleaning up"
msgstr "S'està netejant"

#: ../DistUpgrade/DistUpgradeView.py:363
#, python-format
msgid ""
"%(amount)d installed package is no longer supported by Canonical. You can "
"still get support from the community."
msgid_plural ""
"%(amount)d installed packages are no longer supported by Canonical. You can "
"still get support from the community."
msgstr[0] ""
"%(amount)d paquet instal·lat ja no està mantingut per Canonical. Així i tot, "
"encara podeu obtindre assistència de la comunitat."
msgstr[1] ""
"%(amount)d dels paquets instal·lats ja no estan mantinguts per Canonical. "
"Així i tot, encara podeu obtindre assistència de la comunitat."

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:372
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "Se suprimirà %d paquet."
msgstr[1] "Se suprimiran %d paquets."

#: ../DistUpgrade/DistUpgradeView.py:377
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "S'instal·larà %d paquet nou"
msgstr[1] "S'instal·laran %d paquets nous"

#: ../DistUpgrade/DistUpgradeView.py:383
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "S'actualitzarà %d paquet"
msgstr[1] "S'actualitzaran %d paquets"

#: ../DistUpgrade/DistUpgradeView.py:391
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"Heu de baixar un total de %s. "

#: ../DistUpgrade/DistUpgradeView.py:396
msgid ""
"Installing the upgrade can take several hours. Once the download has "
"finished, the process cannot be canceled."
msgstr ""

#: ../DistUpgrade/DistUpgradeView.py:400
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be canceled."
msgstr ""

#: ../DistUpgrade/DistUpgradeView.py:405
msgid "Removing the packages can take several hours. "
msgstr ""

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:410
msgid "The software on this computer is up to date."
msgstr ""

#: ../DistUpgrade/DistUpgradeView.py:411
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr ""
"No hi ha actualitzacions disponibles per al vostre sistema. L'actualització "
"es cancel·larà"

#: ../DistUpgrade/DistUpgradeView.py:427
msgid "Reboot required"
msgstr "Cal que reinicieu el sistema"

#: ../DistUpgrade/DistUpgradeView.py:428
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr ""
"L'actualització ha finalitzat i cal reiniciar el sistema. Voleu fer-ho ara?"

#: ../DistUpgrade/DistUpgradeView.py:434
#, fuzzy
msgid "Action required"
msgstr "Cal que reinicieu el sistema"

#: ../DistUpgrade/DistUpgradeView.py:435
msgid "Exit all other instances of Ubuntu WSL before continuing."
msgstr ""

#: ../DistUpgrade/DistUpgradeView.py:436
msgid "Unsaved progress may otherwise be lost."
msgstr ""

#: ../DistUpgrade/DistUpgradeView.py:439
#, fuzzy
msgid "WSL restart required"
msgstr "Cal que reinicieu el sistema"

#: ../DistUpgrade/DistUpgradeView.py:440
msgid "Exit this instance of Ubuntu WSL."
msgstr ""

#: ../DistUpgrade/DistUpgradeView.py:441
#, fuzzy
msgid "The upgrade will then be complete."
msgstr "S'ha completat l'actualització del sistema."

#: ../DistUpgrade/DistUpgradeViewText.py:82
#, python-format
msgid "Progress: [%3i%%]"
msgstr "Progrés: [%3i%%]"

#: ../DistUpgrade/DistUpgradeViewText.py:141
msgid ""
"Please report this as a bug and include the files /var/log/dist-upgrade/main."
"log and /var/log/dist-upgrade/apt.log in your report. The upgrade has "
"aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Informeu d'això com un error i adjunteu els fitxers /var/log/dist-upgrade/"
"main.log i /var/log/dist-upgrade/apt.log al vostre informe. S'ha interromput "
"l'actualització.\n"
"S'ha guardat el vostre fitxer sources.list original a /etc/apt/sources.list."
"distUpgrade."

#: ../DistUpgrade/DistUpgradeViewText.py:165
msgid "Aborting"
msgstr "S'està interrompent"

#: ../DistUpgrade/DistUpgradeViewText.py:171
msgid "Demoted:\n"
msgstr "Degradat:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:178
msgid "To continue please press [ENTER]"
msgstr "Premeu la tecla de retorn per continuar"

#: ../DistUpgrade/DistUpgradeViewText.py:215
#: ../DistUpgrade/DistUpgradeViewText.py:253
msgid "Details [d]"
msgstr "Detalls [d]"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:222
#: ../DistUpgrade/DistUpgradeViewText.py:271
msgid "n"
msgstr "n"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:225
msgid "d"
msgstr "d"

#: ../DistUpgrade/DistUpgradeViewText.py:230
#, python-format
msgid "No longer supported: %s\n"
msgstr "Ja no es mantenen: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:235
#, python-format
msgid "Remove: %s\n"
msgstr "Suprimeix: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:245
#, python-format
msgid "Install: %s\n"
msgstr "Instal·la: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:250
#, python-format
msgid "Upgrade: %s\n"
msgstr "Actualitza: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:268
msgid "Continue [Yn] "
msgstr "Continua [Sn] "

#: ../DistUpgrade/DistUpgradeViewText.py:293
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"Cal reiniciar el sistema per a finalitzar l'actualització.\n"
"Si seleccioneu «s», es reiniciarà el sistema."

#: ../DistUpgrade/GtkProgress.py:71
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr "S'està baixant el fitxer %(current)li de %(total)li a %(speed)s/s"

#: ../DistUpgrade/GtkProgress.py:77
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "S'està baixant el fitxer %(current)li de %(total)li"

#: ../data/gtkbuilder/AcquireProgress.ui.h:1
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:7
msgid "Show progress of individual files"
msgstr "Mostra el progrés dels fitxers individuals"

#: ../data/gtkbuilder/DistUpgrade.ui.h:1
msgid "_Cancel Upgrade"
msgstr "_Cancel·la l'actualització"

#: ../data/gtkbuilder/DistUpgrade.ui.h:2
msgid "_Resume Upgrade"
msgstr "_Reprén l'actualització"

#: ../data/gtkbuilder/DistUpgrade.ui.h:3
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>Voleu cancel·lar l'actualització?</big></b>\n"
"\n"
"Si ho feu, pot ser que el sistema siga inutilitzable. És extremament "
"recomanable continuar amb l'actualització."

#: ../data/gtkbuilder/DistUpgrade.ui.h:6
msgid "_Start Upgrade"
msgstr "_Comença l'actualització"

#: ../data/gtkbuilder/DistUpgrade.ui.h:9
msgid "_Replace"
msgstr "_Reemplaça"

#: ../data/gtkbuilder/DistUpgrade.ui.h:10
msgid "Difference between the files"
msgstr "Diferències entre els fitxers"

#: ../data/gtkbuilder/DistUpgrade.ui.h:11
msgid "_Report Bug"
msgstr "_Informa de l'error"

#: ../data/gtkbuilder/DistUpgrade.ui.h:12
msgid "_Continue"
msgstr "_Continua"

#: ../data/gtkbuilder/DistUpgrade.ui.h:13
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>Voleu iniciar l'actualització?</big></b>"

#: ../data/gtkbuilder/DistUpgrade.ui.h:15
msgid ""
"<b><big>Restart the system to complete the upgrade</big></b>\n"
"\n"
"Please save your work before continuing."
msgstr ""
"<b><big>Reinicieu el sistema per a completar l'actualització</big></b>\n"
"\n"
"Guardeu la vostra faena abans de continuar."

#: ../data/gtkbuilder/DistUpgrade.ui.h:18
msgid "Distribution Upgrade"
msgstr "Actualització de la distribució"

#: ../data/gtkbuilder/DistUpgrade.ui.h:20
#, no-c-format
msgid "<b><big>Upgrading Ubuntu to version %s</big></b>"
msgstr ""

#: ../data/gtkbuilder/DistUpgrade.ui.h:21
msgid "    "
msgstr "    "

#: ../data/gtkbuilder/DistUpgrade.ui.h:23
msgid "Setting new software channels"
msgstr "S'estan configurant els canals de programari nous"

#: ../data/gtkbuilder/DistUpgrade.ui.h:25
msgid "Restarting the computer"
msgstr "S'està reiniciant l'ordinador"

#: ../data/gtkbuilder/DistUpgrade.ui.h:28
msgid "Terminal"
msgstr "Terminal"

#: ../data/gtkbuilder/ReleaseNotes.ui.h:2
msgid "_Upgrade"
msgstr "_Actualitza"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:1
msgid "<b>A new version of Ubuntu is available. Would you like to upgrade?</b>"
msgstr ""
"<b>Hi ha disponible una versió nova de l'Ubuntu. Voleu actualitzar el vostre "
"sistema?</b>"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:2
msgid "Don't Upgrade"
msgstr "No actualitzis"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:3
msgid "Ask Me Later"
msgstr "Demana-m'ho més tard"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:4
msgid "Yes, Upgrade Now"
msgstr "Actualitza"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:5
msgid "You have declined to upgrade to the new Ubuntu"
msgstr "Heu triat no actualitzar a la versió nova de l'Ubuntu"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:6
msgid ""
"You can upgrade at a later time by opening Software Updater and click on "
"\"Upgrade\"."
msgstr ""

#: ../data/com.ubuntu.release-upgrader.policy.in.h:1
msgid "Perform a release upgrade"
msgstr ""

#: ../data/com.ubuntu.release-upgrader.policy.in.h:2
msgid "To upgrade Ubuntu, you need to authenticate."
msgstr "Cal que vos autentiqueu per actualitzar l'Ubuntu."

#: ../data/com.ubuntu.release-upgrader.policy.in.h:3
msgid "Perform a partial upgrade"
msgstr ""

#: ../data/com.ubuntu.release-upgrader.policy.in.h:4
msgid "To perform a partial upgrade, you need to authenticate."
msgstr "Cal que vos autentiqueu per fer una actualització parcial."

#: ../do-partial-upgrade:75 ../do-release-upgrade:80
msgid "Show version and exit"
msgstr "Mostra la versió i ix"

#: ../do-partial-upgrade:78 ../do-release-upgrade:87
msgid "Directory that contains the data files"
msgstr "El directori que conté els fitxers de dades"

#: ../do-partial-upgrade:81 ../do-release-upgrade:100
msgid "Run the specified frontend"
msgstr "Executa el frontal especificat"

#: ../do-partial-upgrade:112
msgid "Running partial upgrade"
msgstr "S'està executant una actualització parcial"

#: ../do-release-upgrade:34 ../do-release-upgrade:46
msgid "Downloading the release upgrade tool"
msgstr "S'està baixant l'eina d'actualització a una versió nova"

#: ../do-release-upgrade:83
msgid ""
"If using the latest supported release, upgrade to the development release"
msgstr ""

#: ../do-release-upgrade:90 ../check_new_release_gtk.py:188
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr ""
"Proveu d'actualitzar a l'última versió fent servir l'actualitzador de "
"$distro-proposed"

#: ../do-release-upgrade:94
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""
"Executeu un mode d'actualització especial.\n"
"Actualment disposeu del mode «desktop» per a actualitzacions de sistemes "
"d'escriptori i del mode «server» per a sistemes servidors."

#: ../do-release-upgrade:103
msgid ""
"Check only if a new distribution release is available and report the result "
"via the exit code"
msgstr ""
"Només verifica si hi ha disponible una versió nova i informa del resultat "
"mitjançant el codi d'eixida"

#: ../do-release-upgrade:108
msgid ""
"Try the upgrade with third party mirrors and repositories enabled instead of "
"commenting them out."
msgstr ""

#: ../do-release-upgrade:114
msgid ""
"A comma-separated list of environment variables (e.g. VAR1=VALUE1,"
"VAR2=VALUE2) that should be set during the upgrade."
msgstr ""

#: ../do-release-upgrade:126
msgid "The options --devel-release and --proposed are"
msgstr "Les opcions --devel-release i --proposed són"

#: ../do-release-upgrade:127
msgid "mutually exclusive. Please use only one of them."
msgstr "mútuament excloents. Feu servir només una d'elles."

#: ../do-release-upgrade:131
msgid "Checking for a new Ubuntu release"
msgstr ""

#: ../do-release-upgrade:143
msgid "Your Ubuntu release is not supported anymore."
msgstr "La vostra versió de l'Ubuntu ja no està mantinguda."

#: ../do-release-upgrade:144
msgid ""
"For upgrade information, please visit:\n"
"%(url)s\n"
msgstr ""
"Per obtindre més informació sobre actualitzacions, aneu a:\n"
"%(url)s\n"

#: ../do-release-upgrade:159
msgid "In /etc/update-manager/release-upgrades Prompt "
msgstr ""

#: ../do-release-upgrade:160
msgid "is set to never so upgrading is not possible."
msgstr ""

#: ../do-release-upgrade:162
msgid "There is no development version of an LTS available."
msgstr ""

#: ../do-release-upgrade:163
msgid "To upgrade to the latest non-LTS development release "
msgstr ""

#: ../do-release-upgrade:164
msgid "set Prompt=normal in /etc/update-manager/release-upgrades."
msgstr ""

#: ../do-release-upgrade:166
msgid "Upgrades to the development release are only "
msgstr ""

#: ../do-release-upgrade:167
msgid "available from the latest supported release."
msgstr ""

#: ../do-release-upgrade:169
msgid "No new release found."
msgstr ""

#: ../do-release-upgrade:174
msgid "Release upgrade not possible right now"
msgstr "En estos moments no es pot efectuar l'actualització de versió"

#: ../do-release-upgrade:175
#, c-format
msgid ""
"The release upgrade can not be performed currently, please try again later. "
"The server reported: '%s'"
msgstr ""
"En estos moments no es pot efectuar l'actualització de versió. Torneu-ho a "
"provar més tard. El servidor ha respost: «%s»"

#: ../do-release-upgrade:181
#, c-format
msgid "New release '%s' available."
msgstr "La versió nova «%s» ja està disponible"

#: ../do-release-upgrade:182
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "Executeu «do-release-upgrade» per actualitzar a la nova versió."

#: ../do-release-upgrade:199
msgid "Please install all available updates for your release before upgrading."
msgstr ""

#: ../do-release-upgrade:215
msgid ""
"You have not rebooted after updating a package which requires a reboot. "
"Please reboot before upgrading."
msgstr ""

#: ../check_new_release_gtk.py:88
msgid "Software Updater"
msgstr "Gestor d'actualitzacions"

#: ../check_new_release_gtk.py:120
#, python-format
msgid "Ubuntu %(version)s Upgrade Available"
msgstr "Hi ha disponible una actualitzacio de l'Ubuntu %(version)s"

#. print("don't upgrade")
#: ../check_new_release_gtk.py:151
#, python-format
msgid "You have declined the upgrade to Ubuntu %s"
msgstr "Heu triat no actualitzar a l'Ubuntu %s"

#: ../check_new_release_gtk.py:184
msgid "Check if upgrading to the latest devel release is possible"
msgstr ""
"Comprova si és possible actualitzar a la darrera versió de desenvolupament"

#: ../check_new_release_gtk.py:193
msgid "Add debug output"
msgstr ""

#~ msgid "Could not calculate sources.list entry"
#~ msgstr "No s'ha pogut calcular l'entrada del fitxer sources.list"

#~ msgid ""
#~ "Unable to locate any package files, perhaps this is not a Ubuntu Disc or "
#~ "the wrong architecture?"
#~ msgstr ""
#~ "No s'ha pogut trobar cap paquet, esteu utilitzat un disc de l'Ubuntu per "
#~ "a l'arquitectura adequada?"

#~ msgid "Failed to add the CD"
#~ msgstr "No s'ha pogut afegir el CD"

#~ msgid ""
#~ "There was a error adding the CD, the upgrade will abort. Please report "
#~ "this as a bug if this is a valid Ubuntu CD.\n"
#~ "\n"
#~ "The error message was:\n"
#~ "'%s'"
#~ msgstr ""
#~ "S'ha produït un error en afegir el CD i l'actualització s'ha cancel·lat. "
#~ "Informeu d'este error si això ha passat amb un CD d'Ubuntu vàlid.\n"
#~ "\n"
#~ "El missatge d'error ha estat:\n"
#~ "«%s»"

#~ msgid "Include latest updates from the Internet?"
#~ msgstr "Voleu incloure les darreres actualitzacions d'Internet?"

#~ msgid ""
#~ "The upgrade system can use the internet to automatically download the "
#~ "latest updates and install them during the upgrade.  If you have a "
#~ "network connection this is highly recommended.\n"
#~ "\n"
#~ "The upgrade will take longer, but when it is complete, your system will "
#~ "be fully up to date.  You can choose not to do this, but you should "
#~ "install the latest updates soon after upgrading.\n"
#~ "If you answer 'no' here, the network is not used at all."
#~ msgstr ""
#~ "El sistema d'actualitzacions pot utilitzar Internet per baixar i "
#~ "instal·lar automàticament les darreres actualitzacions. Si disposeu d'una "
#~ "connexió de xarxa, esta opció és molt recomanable.\n"
#~ "\n"
#~ "L'actualització trigarà més, però en acabar tindreu un sistema "
#~ "completament actualitzat. Podeu optar per no fer-ho, però en breu haureu "
#~ "d'actualitzar el sistema.\n"
#~ "Si responeu «no» la xarxa no s'utilitzarà per a res."

#~ msgid "Failed to add the cdrom"
#~ msgstr "No s'ha pogut afegir el CD-ROM"

#~ msgid "Sorry, adding the cdrom was not successful."
#~ msgstr "No s'ha pogut afegir el CD-ROM correctament."

#~ msgid "Use the given path to search for a cdrom with upgradable packages"
#~ msgstr ""
#~ "Utilitza el camí especificat per cercar un CDROM amb paquets "
#~ "actualitzables"

#~ msgid ""
#~ "An unresolvable problem occurred while calculating the upgrade:\n"
#~ "%s\n"
#~ "\n"
#~ " This can be caused by:\n"
#~ " * Upgrading to a pre-release version of Ubuntu\n"
#~ " * Running the current pre-release version of Ubuntu\n"
#~ " * Unofficial software packages not provided by Ubuntu\n"
#~ "\n"
#~ msgstr ""
#~ "S'ha produït un problema irresoluble mentre es calculava "
#~ "l'actualització:\n"
#~ "%s\n"
#~ "\n"
#~ "Això pot ser degut a:\n"
#~ " * l'actualització a una versió en desenvolupament de l'Ubuntu\n"
#~ " * l'execució de l'actual versió en desenvolupament de l'Ubuntu\n"
#~ " * la instal·lació de paquets no oficials de programari i no "
#~ "proporcionats per l'Ubuntu\n"
#~ "\n"

#~ msgid "Sandbox setup failed"
#~ msgstr "Ha fallat la configuració d'un entorn de proves"

#~ msgid "It was not possible to create the sandbox environment."
#~ msgstr "No ha estat possible crear un entorn de proves."

#~ msgid "Sandbox mode"
#~ msgstr "Mode de prova"

#~ msgid ""
#~ "Your python install is corrupted. Please fix the '/usr/bin/python' "
#~ "symlink."
#~ msgstr ""
#~ "La vostra instal·lació del Python està malmesa. Reviseu l'enllaç simbòlic "
#~ "«/usr/bin/python»."

#~ msgid "Package 'debsig-verify' is installed"
#~ msgstr "El paquet «debsig-verify» està instal·lat"

#~ msgid ""
#~ "The upgrade can not continue with that package installed.\n"
#~ "Please remove it with synaptic or 'apt-get remove debsig-verify' first "
#~ "and run the upgrade again."
#~ msgstr ""
#~ "L'actualització no pot continuar amb este paquet instal·lat.\n"
#~ "Elimineu-lo amb el Synaptic o amb l'orde «apt-get remove debsig-verify» i "
#~ "torneu a executar l'actualització."

#~ msgid ""
#~ "While scanning your repository information no mirror entry for the "
#~ "upgrade was found. This can happen if you run a internal mirror or if the "
#~ "mirror information is out of date.\n"
#~ "\n"
#~ "Do you want to rewrite your 'sources.list' file anyway? If you choose "
#~ "'Yes' here it will update all '%s' to '%s' entries.\n"
#~ "If you select 'No' the upgrade will cancel."
#~ msgstr ""
#~ "No s'ha trobat cap entrada de rèplica per a l'actualització a la vostra "
#~ "informació sobre els dipòsits. Això pot ésser degut a què estigueu "
#~ "executant una rèplica interna o que les dades de les rèpliques no "
#~ "estiguen actualitzades.\n"
#~ "\n"
#~ "Voleu sobreescriure el fitxer «sources.list» de totes maneres? Si trieu "
#~ "que «Sí», totes les entrades «%s» s'actualitzaran a «%s». Si trieu que "
#~ "«No», es cancel·larà l'actualització."

#~ msgid ""
#~ "The upgrade has aborted. The upgrade needs a total of %s free space on "
#~ "disk '%s'. Please free at least an additional %s of disk space on '%s'. "
#~ "Empty your trash and remove temporary packages of former installations "
#~ "using 'sudo apt-get clean'."
#~ msgstr ""
#~ "S'ha interromput l'actualització. Cal un total de %s d'espai lliure al "
#~ "disc «%s», per la qual cosa hauríeu d'alliberar un mínim de %s d'espai "
#~ "addicional al disc «%s». Buideu la paperera i suprimiu els paquets "
#~ "temporals d'anteriors instal·lacions utilitzant l'orde «sudo apt-get "
#~ "clean»."

#~ msgid "evms in use"
#~ msgstr "S'està utilitzant l'evms"

#~ msgid ""
#~ "Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
#~ "software is no longer supported, please switch it off and run the upgrade "
#~ "again when this is done."
#~ msgstr ""
#~ "El vostre sistema utilitza el gestor de volum «evms» a /proc/mounts. Este "
#~ "programari no es mantindrà més, desconnecteu-lo i torneu a executar "
#~ "l'actualització."

#~ msgid ""
#~ "This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
#~ "version of this driver is available that works with your video card in "
#~ "Ubuntu 10.04 LTS.\n"
#~ "\n"
#~ "Do you want to continue?"
#~ msgstr ""
#~ "Este ordinador utilitza el controlador gràfic «nvidia» d'NVIDIA. No hi ha "
#~ "cap versió disponible d'este controlador que funcione amb el vostre "
#~ "maquinari a l'Ubuntu 10.04 LTS.\n"
#~ "\n"
#~ "Voleu continuar?"

#~ msgid "No i686 CPU"
#~ msgstr "Sense CPU i686"

#~ msgid ""
#~ "Your system uses an i586 CPU or a CPU that does not have the 'cmov' "
#~ "extension. All packages were built with optimizations requiring i686 as "
#~ "the minimal architecture. It is not possible to upgrade your system to a "
#~ "new Ubuntu release with this hardware."
#~ msgstr ""
#~ "El vostre sistema utilitza una CPU i586 o una CPU que no té l'extensió "
#~ "«cmov». Tots els paquets s'han construït amb optimitzacions que "
#~ "requereixen un i686 com a arquitectura mínima. No és possible actualitzar "
#~ "el sistema a una versió nova de l'Ubuntu amb este maquinari."

#~ msgid "Sandbox upgrade using aufs"
#~ msgstr "Actualització en entorn de proves utilitzant aufs"

#~ msgid "Test upgrade with a sandbox aufs overlay"
#~ msgstr "Prova d'actualitzar en un entorn de proves amb aufs"

#~ msgid "No new release found"
#~ msgstr "No s'ha trobat cap versió nova"

#~ msgid ""
#~ "Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
#~ "desktop or edubuntu-desktop package and it was not possible to detect "
#~ "which version of Ubuntu you are running.\n"
#~ " Please install one of the packages above first using synaptic or apt-get "
#~ "before proceeding."
#~ msgstr ""
#~ "No teniu instal·lats el paquet ubuntu-desktop, kubuntu-desktop, xubuntu-"
#~ "desktop o edubuntu-desktop i no s'ha pogut detectar la versió de l'Ubuntu "
#~ "que esteu utilitzant.\n"
#~ "Instal·leu un d'aquests paquets abans de continuar; per fer-ho podeu "
#~ "utilitzar el Synaptic o l'apt-get"

#~ msgid ""
#~ "An unresolvable problem occurred while calculating the upgrade.\n"
#~ "\n"
#~ " This can be caused by:\n"
#~ " * Upgrading to a pre-release version of Ubuntu\n"
#~ " * Running the current pre-release version of Ubuntu\n"
#~ " * Unofficial software packages not provided by Ubuntu\n"
#~ "\n"
#~ msgstr ""
#~ "S'ha produït un problema irresoluble mentre es feia el càlcul per a "
#~ "l'actualització.\n"
#~ "\n"
#~ " Això pot ser degut a:\n"
#~ " * l'actualització a una versió en desenvolupament de l'Ubuntu\n"
#~ " * l'execució de l'actual versió en desenvolupament de l'Ubuntu\n"
#~ " * la instal·lació de paquets no oficials de programari no proveïts per "
#~ "l'Ubuntu\n"
#~ "\n"
